angular-ngc/karma.bzl

```diff
@@ -0,0 +1,62 @@
+def _to_manifest_path(ctx, file):
```

Comment:
Lets move this file into the tools directory where I'm starting to move most of the .bzl files

Reply:
done